person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+end
```

Comment:
- [ ] kindly make name age and id accessible. like this ` attr_accessor :name, :age `,  ``attr_reader :id``